kafka/consumer-group/consumer_group.go

```diff
@@ -340,7 +352,15 @@ func (cg *ConsumerGroup) topicConsumer(topic string, messages chan<- *sarama.Con
 	// Consume all the assigned partitions
 	var wg sync.WaitGroup
 	for _, pid := range myPartitions {
-
+		//Create PartitionInfo and send it for ingestion through the partition channel
+		//In case of re-balancing this function will be triggered again and the latest information will be sent
+
+		metricName := name + "." + topic + "." + cg.instance.ID + "." + time.Now().Format(time.RFC850)
```

Comment:
why are we adding timestamp ? that is part of metrics metadata and what are we trying to monitor here?

Reply:
We are trying to monitor the partitions owned by Dumx. And the timestamp is to make the key unique else the metric value would be overwritten

Reply:
I don't recommend this, in this manner we will be new key per millisecond (based on the unit of `time.Now`. If you understand the metrics, they're meant to change and reflect current status of app. The historical value is something that will be given by store where metrics are ingested. Please remove timestamp.